sequence_processing_pipeline/NuQCJob.py

```diff
@@ -94,7 +93,6 @@ def __init__(self, fastq_root_dir, output_path, sample_sheet_path,
         self.qiita_job_id = qiita_job_id
         self.suffix = 'fastq.gz'
         self.movi_path = movi_path
-        self.gres_value = gres_value
```

Comment:
can you bring this back? I'm still not 100% if it's safe to completely remove it.